ddpui/models/alert.py

```
@@ -0,0 +1,178 @@
+    message_placeholders = models.JSONField(default=list)
+
+    # Status
+    is_active = models.BooleanField(default=True)
+
+    # Timestamps
+    created_at = models.DateTimeField(auto_created=True, default=timezone.now)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        db_table = "alert"
+        ordering = ["-updated_at"]
```

Ruff (0.15.10)

[warning] Mutable default value for class attribute

(RUF012)